src/cascadia/TerminalControl/TermControl.idl

```diff
@@ -45,6 +45,20 @@ namespace Microsoft.Terminal.TerminalControl
         String Uri { get; };
     }
 
+    enum NoticeLevel
+    {
+        Debug = 10,
+        Info = 20,
+        Warning = 30,
+        Error = 40,

```

Comment:
Do these values have a particular meaning?

Reply:
No, I just pulled 4 levels out of my butt and left space for future ones to be put between without renumbering, like BASIC used to do...

If you have another idea... lemme know.